api/integrations/launch_darkly/tests/data/standard_environments_response.json

```diff
@@ -0,0 +1,102 @@
+[
```

Comment:
General comment that I'd rather move these tests to `api/tests/`

Reply:
ok sure, will do